src/browser/api/notifications/shapes.ts

```diff
@@ -31,6 +31,10 @@ export interface NotificationMessage {
     data: any;
 }
 
+export interface NoteId extends Identity {
```

Comment:
Back to the naming of this interface, are we sure the interface is not defining the shape of a `Note` and not the shape of a `NoteId`?

Also, are we consistently abbreviating identity / identification as `Id` in extended implementations?

Reply:
🌳